app/api/items/search_for_items.feature

```diff
@@ -36,12 +36,12 @@ Feature: Search for items
       | item_id | language_tag | title                                   |
       | 1       | fr           | (the) Our Chapter                       |
       | 2       | fr           | (the) Our Task ___                      |
-      | 3       | en           | (the) Our Course                        |
+      | 3       | en           | (the) Our Task                          |
       | 4       | fr           | (the) \|\|\|Our Skill \\\\\\%\\\\%\\ :) |
       | 6       | en           | Another Chapter                         |
       | 6       | fr           | Un autre chapitre                       |
       | 7       | en           | Another %%%Task                         |
-      | 8       | en           | Another %%%Course                       |
+      | 8       | en           | Another %%%Task 2                       |
```

Comment:
I suppose that was the kind of data which was useful only to test both the task and the course case... so testing twice the tasks do not make sense anymore.

Reply:
This one was indeed only used for Courses. For the other ones it's hard to tell.